Lib/mimetypes.py

```diff
@@ -23,11 +23,6 @@
 read_mime_types(file) -- parse one file, return a dictionary or None
 """
 
-import os
```

Comment:
I wonder if lazy import of `os` module is worth it here, it is needed three times and the improvement is small and only in a special case.

Reply:
Python always imports `os` and `sys` at startup. Moving `import sys` makes sense, it's only used by _main(). But I'm not sure about moving `os`.

Reply:
In the OP, @hugovk provides timings for when the python is run without the `site.py`  module (`python -S`), in which case `os` and `sys` are not loaded I believe?

Reply:
Yes, it makes a difference with `python -S`, compare the last three images above.

But I'm fine reverting any of these, because I expect with `site.py` is by far the most usual thing.